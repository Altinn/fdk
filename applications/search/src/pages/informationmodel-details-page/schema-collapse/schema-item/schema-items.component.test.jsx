import React from 'react';
import { shallow } from 'enzyme';
import { SchemaItem } from './schema-items.component';

test('should render SchemaItem with props', () => {
  const defaultProps = {
    title: 'Tittel',
    type: 'Type',
    marginLeft: '16'
  };
  const wrapper = shallow(<SchemaItem {...defaultProps} />);
  expect(wrapper).toMatchSnapshot();
});
