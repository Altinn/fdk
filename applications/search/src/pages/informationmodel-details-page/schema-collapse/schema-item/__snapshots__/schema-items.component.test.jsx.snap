// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SchemaItem with props 1`] = `
<div
  className="d-flex schema-collapse--item"
  name="Tittel"
  style={
    Object {
      "marginLeft": "16",
    }
  }
>
  <div
    className="w-50 pl-0"
  >
    Tittel
  </div>
  <div
    className="w-50"
  >
    Type
  </div>
</div>
`;
